proposals/drafts/ocdid_governance.rst

```diff
@@ -0,0 +1,113 @@
+
+Commits should be reviewed within 2 weeks of a pull request. Accellerated timeline needs should be communicated via Slack/email.
+
+* If, after two months there is neither a formal approval or an ongoing conversation around a request, that request will be considered accepted.
+* If a conversation around a request cannot reach consensus, after two months the ``contributor`` may request a final vote from the cohort of existing ``committers``.
+
+Approval will be handled on a per-file, rather than a per-commit, basis.
+
+New commits should include have a well-formed explanations, especially if generating new IDs or types.
+
+Formalized guidelines for approval will be created for use by ``committers``. These guidelines will include:
+
+* Syntax check guidelines
+* Type check guidelines and a glossary of existing types
+* Dupe check guidelines
+* Instructions for including and checking for correct sameAs aliasing
+* Differentiating standards of review between generative/corrective requests
+
+Government contributions
+~~~~~~~~~~~~~~~~~~~~~~~~
+Should a government entity wish to contribute to the repository, they will initially be asked to work directly with an existing ``committer`` to prepare to integrate their identifier set.
+
+* A second ``committer`` is still required for approving government contributions, even if a government contributor becomes a ``committer`` themselves.
+* Even so, government contributors should be given wide deference within their geographic area.
+* If, because of naming conventions, geographic edge cases, etc. a government contributor requests a deviation from existing OCDID nomenclature, the community will attempt to reasonably accommodate that request (e.g. using “police_jury” as a type in lieu of “county_council” for Louisiana's county legislative body)
+
+Government entities (and the ``committers`` they work with) will be expected to reconcile and appropriately alias these cases with existing OCDIDs in order to ensure maximum compatibility.
+
+Identifiers created by government officials that are used in official data will be marked as such in the repository, so that developers can quickly identify the preferred identifier in case of conflict. Caution should be used, and the orginal submitter consulted with if possible, before changing government submitted identifiers.
+
+A section of documentatoin specifically aimed at government staff will be created, where they can learn more about the project and how to get involved, as well as how to reach out to the community to get help.
+
+Support
+--------
+``Committers`` will be required to participate in a quarterly review of new OCDIDs in order to ensure quality on-going.
+``Committers`` will be requested to contribute and maintain an ongoing style guide for creating new district types.
+``Committers`` will be required to participate in > 60% of all formal votes/actions as announced.
```

Comment:
I think we need there to be geographic segmentation of committers. A committer for Antarctica shouldn't have to participate in US votes/actions.

Reply:
agree.  I think updating this document w/ committers and their jurisdiction would be useful, perhaps ~3 tiers

- a few people might be considered 'global' and their participation could be requested in all votes (but we'd presumably at least informally waive the voting requirement?)
- most people would be scoped to a country
- any gov't officials would probably be scoped to their jurisdiction and thus not expected to participate in wider conversations

Reply:
Completely agree on the need for geographic segmentation, and like the tiers. Also adding in a caveat about flexibility of process/governance at a country level.